graphs/util/db.py

```diff
@@ -1608,6 +1608,300 @@ def find_all_graphs_containing_nodes(uid, search_type, search_word, view_type, d
 
 	return graph_dict.values()
 
+
+def uploadGPMLFile(username, graphJSON, title):
```

Comment:
Can you rename this method to lowercase  with words separated by underscores ? I know a lot of existing methods do not follow the PEP8 naming convention, but going ahead we should strictly follow PEP8 rules.